index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Portifolio | Stanley</title>
+</head>
+<body>
+  <!--Navigation bar--> 
+    <header> 
+     <div class= "header-logo">
+      <div class="bar-card" id="myNavbar">
+        <a href="#home" class="logo">Ezeoke Stanley</a> 
+  <!--Right-sided navbar links-->
+        <nav>
+        <div class="right-bar">
+          <img src="./icons/menu.png" alt="envelope icon" class="envelope">
+        </div>
+      </nav>
+      </div>
+    </div>
+  </header>
```

Comment:
[ Optional ]
- [ ] It would be nice you adjust a few things about the way your structure the header. You are using too many `div`s and that's giving you issues to style. I suggest you make it simple just as the previous reviewer suggested.
- More Hint: 👍 
```
<header>
     <a href="#home" class="logo">Ezeoke Stanley</a>
     <nav>
          <img src="./icons/menu.png" alt="envelope icon" class="envelope">
      </nav>
</header>
```
Please try that and style the header to match the one in the Figma design